lib/node_modules/@stdlib/stats/base/snanmeanwd/test/test.ndarray.native.js

```diff
@@ -175,9 +169,8 @@ tape( 'the function supports an `offset` parameter', opts, function test( t ) {
 		NaN,
 		NaN   // 4
 	]);
-	N = floor( x.length / 2 );
 
-	v = snanmeanwd( N, x, 2, 1 );
+	v = snanmeanwd( 4, x, 2, 1 );
```

Comment:
```suggestion
	v = snanmeanwd( 5, x, 2, 1 );
```
see how the array is being traversed by following the commented indexes